controllers/usersignup/usersignup_controller_test.go

```diff
@@ -4430,6 +4430,42 @@ func TestUserSignupStatusNotReady(t *testing.T) {
 
 	})
 
+	// If a space is updating, don't mark usersignups as "Incomplete", since
+	// they'll get rejected by the registration service
```

Comment:
"Rejected" word can be a bit confusing here. The usersignup is not being rejected per se. It will be marked as "Incomplete" which can cause issues like the kube API proxy requests being rejected or wrong user signup status in UI (pending provisioning instead of ready).

Reply:
I've updated these tests in both 97d45fe and 583f076, please re-review.